Backpack/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarMonth.kt

```diff
@@ -41,7 +41,7 @@ internal inline fun CalendarMonth(
   locale: Locale,
   monthsTextStyle: TextStyle,
   weekFields: WeekFields,
-  footers: List<YearMonth>,
+  footers: Set<YearMonth>,
```

Comment:
wouldn't we want the footers be a list to have an order?

Reply:
we are creating cells for each footer in `CalendarMonth`, though, so it is being used as a list, rather than a set